src/h5web/vis-packs/core/shared/PanZoomMesh.tsx

```diff
@@ -2,25 +2,38 @@ import { useRef, useCallback, useEffect } from 'react';
 import type { Vector3 } from 'three';
 import { useThree } from '@react-three/fiber';
 import { clamp } from 'lodash';
-import { useWheelCapture } from '../hooks';
+import { useCanvasScales, useWheelCapture } from '../hooks';
 import type { ThreeEvent } from '@react-three/fiber/dist/declarations/src/core/events';
+import type { Domain } from '../models';
 
 const ZOOM_FACTOR = 0.95;
 
-function PanZoomMesh() {
+interface Props {
+  abscissaDomain: Domain;
+  ordinateDomain: Domain;
+}
+
+function PanZoomMesh(props: Props) {
+  const { abscissaDomain, ordinateDomain } = props;
```

Comment:
I need the original domains to compute the width and height of the mesh, like in `Mesh`.